tests/unit/test_collection_queries.py

```diff
@@ -5,14 +5,19 @@
 from earthaccess.search import DataCollections
 
 valid_single_dates = [
-    ("2001-12-12", "2001-12-21", "2001-12-12T00:00:00Z,2001-12-21T00:00:00Z"),
+    ("2001-12-12", "2001-12-21", "2001-12-12T00:00:00Z,2001-12-21T23:59:59Z"),
```

Comment:
:thinking: is it worth adding some non-utc timestamps like `'2024-04-18T22:13:21.469360-08:00'`?

Reply:
Added a utc offset test.